.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
+
+-   [ ] Tests for the changes have been added (for bug fixes / features)
+-   [ ] Examples have been added / updated (for bug fixes / features)
+
+## PR Type
+
+What kind of change does this PR introduce?
+
+<!-- Please check the one that applies to this PR using "x". -->
+
+-   [ ] Bugfix
+-   [ ] Feature
+-   [ ] Code style update (formatting, local variables)
+-   [ ] Refactoring (no functional changes, no api changes)
+-   [ ] Build related changes
+-   [ ] CI related changes
+-   [ ] Documentation content changes
+-   [ ] Example website changes
+-   [ ] Other... Please describe:
+
+## What does this change do?
+
+## What testing did you do?
```

Comment:
What manual testing did you do?

We want to stress that we want clear steps of manual testing because they help:

* The reviewer understand that enough test cases were covered
* Future developers who reference this issue when they modify the same area of the code